app/_kong_plugins/jwt-signer/examples/validate-access-token-issuers.yaml

```
@@ -0,0 +1,15 @@
+description: Validates the not bofore claim in an access token.
+
```

```suggestion
extended_description: |
  Validates the "not before" (`nbf`) claim in an access token.
```
Adding extended description so that we can use the claim name here.